src/main/java/com/jcabi/manifests/Manifests.java

```diff
@@ -162,6 +163,31 @@ public Manifests(final Map<String, String> attrs) {
         this.attributes = new ConcurrentHashMap<String, String>(attrs);
     }
 
+    /**
+     * Reads all values with specified key from several manifests.
+     * Result collections contains all values with specified key
+     * across manifests. Result is empty collection if there is no
+     * such key in any manifests.
+     * @param manifests Manifests
+     * @param key Value key
+     * @return Collections of values
+     * @throws IOException If some I/O problem inside
+     */
+    @SuppressWarnings("PMD.AvoidInstantiatingObjectsInLoops")
+    public static Collection<String> all(final Mfs manifests, final String key)
+        throws IOException {
+        final Collection<String> values = new ArrayList<String>(0);
```

Comment:
@nesterov-n what's the point in having list with 0 initial capacity? We should have at least a few places to begin with